Makefile

```
@@ -115,9 +115,12 @@ explaintest: server_check
 ddltest:
 	@cd cmd/ddltest && $(GO) test -o ../../bin/ddltest -c
 
+CLEAN_UT_BINARY := find . -name '*.test.bin'| xargs rm -rf
```

```suggestion
CLEAN_UT_BINARY := find . -name '*.test.bin'| xargs rm
```
IMHO `rm` should be enough...`rm -rf` is a little scary in a script.

---

Done ... 
I never 'rm -rf /'
:lol